sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/models/ParticipantsCollection.java

```diff
@@ -1,6 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
 
 package com.azure.communication.rooms.models;
```

Comment:
Will need to remove this in the models folder and since we are no longer directly returning ParticipantsCollection.